Zend/zend_globals.h

```diff
@@ -181,7 +181,11 @@ struct _zend_executor_globals {
 
 	JMP_BUF *bailout;
 
-	int error_reporting;
+	int  error_reporting;
+
+	int  fatal_error_backtraces;
+	zval error_backtrace;
```

Comment:
I don't think the alignment provides value here (especially since the `error_reporting` is separated by an empty line) and just unnecessarily touches existing code, breaking `git blame`.

Reply:
agreed, I hadn't meant to align this.